ui/e2e/Dockerfile.sauceconnect

```diff
@@ -0,0 +1,14 @@
+FROM ubuntu
```

Comment:
Wanted to make this `alpine` based, but I couldn't get it to work as it's seemingly Oracle Java 8 based and suffers glibc linking issues documented in: https://github.com/gliderlabs/docker-alpine/blob/master/docs/caveats.md#incompatible-binaries 😢